<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RequeteForum</class>
 <widget class="QWidget" name="RequeteForum">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>239</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Requêtes sur table Forum</string>
  </property>
  <widget class="QLabel" name="resultatLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="departementCheckBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Département</string>
   </property>
  </widget>
  <widget class="QPushButton" name="validerButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>190</y>
     <width>93</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Valider</string>
   </property>
  </widget>
  <widget class="QPushButton" name="annulerButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>190</y>
     <width>93</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
  </widget>
  <widget class="QComboBox" name="anneeComboBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Année du forum</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2009</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2010</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2011</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2012</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="departementSpinBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>55</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>96</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
